src/Models/TreeItems/HelpTreeItem.ts

```diff
@@ -0,0 +1,80 @@
+import {AzExtParentTreeItem, AzExtTreeItem, IActionContext} from "@microsoft/vscode-azext-utils";
+import {ThemeIcon} from "vscode";
+import {OpenUrlTreeItem} from "./OpenUrlTreeItem";
+
+export class HelpTreeItem extends AzExtParentTreeItem {
+  public label: string = "help";
+  public contextValue: string = "help";
+
+  private values: AzExtTreeItem[];
+
+  public constructor(parent: AzExtParentTreeItem | undefined) {
+    super(parent);
+    this.values = [
+      this.readGettingStartedGuidTreeItem,
+      this.readDocumentationTreeItem,
+      this.getCodeSamplesExamplesTreeItem,
+      this.reportAnIssueTreeItem,
+      this.communityAndSupportTreeItem,
+    ];
+  }
+  public async loadMoreChildrenImpl(_clear: boolean, _ctx: IActionContext): Promise<AzExtTreeItem[]> {
+    return this.values;
+  }
+  public hasMoreChildrenImpl(): boolean {
+    return false;
+  }
+
+  public compareChildrenImpl(item1: AzExtTreeItem, item2: AzExtTreeItem): number {
+    // default sorting is based on the label which is being displayed to user.
+    // use id to control the order being dispalyed
+    return item1!.id!.localeCompare(item2!.id!);
+  }
+
+  private get readGettingStartedGuidTreeItem(): AzExtTreeItem {
+    return new OpenUrlTreeItem(
+      this,
+      "0",
+      "Getting Started Guide",
```

Comment:
In the future, we can put all these texts into a json file to simplify. What do you think?

Reply:
yes, I think once this is all happy we can move to a single spot for constants used in UI, internal etc.